starter-code/index.html

```diff
@@ -3,40 +3,87 @@
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
+  <header>
+    <div class="navbar">
+      <img class="logo" src="images/spotify-logo.png" alt="Spotify logo">
+      <nav>
+      <ul class="navbar-links">
+        <li><a href="#">Premium</a></li>
+        <li><a href="#">Discover</a></li>
+        <li><a href="#">Help</a></li>
+        <li><a href="#">Download</a></li>
+      </ul>
+      </nav>
+    </div>
+  </header>
+
+  <div class="splash">
+    <div class="splash-wrapper">
+      <h1>Music for everyone.</h1>
+      <p>Spotify is now free on mobile, tablet and computer.</p>
+      <p>Listen to the right music, wherever you are.</p>
+    </div>
+  </div>
+
+  <div class="whats-on-spotify">
+    <div class="whats-on-spotify-wrapper">
+      <div class="whats-on-spotify-header-wrapper">
+        <h2>What’s on Spotify?</h2>
+      </div>
+      
+      <div class="whats-on-spotify-box-wrapper">
+        <div class="col-3">
+          <img class="icon" src="images/music-icon.png" alt="Music icon">
+          <h3>Millions of Songs</h3>
+          <div class="box-text">
+            <p>There are millions of songs on Spotify</p>
+          </div>
+        </div>
+
+        <div class="col-3">
+          <img class="icon" src="images/high-quality-icon.png" alt="High quality icon">
+          <h3>HD Music</h3>
+          <div class="box-text">
+            <p>Listen to music as if you were listening live</p>
+          </div>
+        </div>
+
+        <div class="col-3">
+          <img class="icon" src="images/devices-icon.png" alt="Devices icon">
+          <h3>Stream Everywhere</h3>
+          <div class="box-text">
+            <p>Stream music on your smartphone, tablet or computer</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="kanye-west">
+    <div class="kanye-west-wrapper">
+      <h2>It’s as yeezy as Kanye West.</h2>
+
+      <div class="search-browse-discover">
+        <h3>Search</h3>
+        <p>Know what you want to listen to? Just search and hit play.</p>
+      </div>
+
+      <div class="search-browse-discover">
+          <h3>Browse</h3>
+          <p>Check out the latest charts, brand new releases and great playlists for right now.</p>
+      </div>
+
+      <div class="search-browse-discover">
+          <h3>Discover</h3>
+          <p>Enjoy new music every Monday with your own personal playlist. Or sit back and enjoy Radio.</p>
+      </div>
+
+      <img class="screen-capture" src="images/spotify-app.jpg" alt="kanye west">
```

Comment:
image should show up on the left side of this div with text on the right. having a logo in between them.